tracing-api/src/main/java/com/palantir/tracing/api/TraceHttpHeaders.java

```diff
@@ -30,6 +30,8 @@
     String SPAN_ID = "X-B3-SpanId";
     String IS_SAMPLED = "X-B3-Sampled"; // Boolean (either “1” or “0”, can be absent)
 
+    String FOR_USER_AGENT = "ForUserAgent";
```

Comment:
nit: Let's match the proxy-style `For-*` style, and standard `User-Agent`:
```suggestion
    String FOR_USER_AGENT = "For-User-Agent";
```